### PR TITLE
Add docs about installation with notification-orm-pack

### PR DESCRIPTION
**Pull request by covex-nn**
I am targeting this branch, because docs.

## Subject

After creating recipe for `sonata-project/notification-orm-pack` installation docs should be updated.

### CODE REVIEW

File: docs/reference/installation.rst [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
-To begin, add the dependent bundles:
+Retrieve the bundle with composer:
 
 .. code-block:: bash
 
     composer require sonata-project/notification-bundle
+
+
+Or if you wish to use doctrine backend, execute ``composer require sonata-project/notification-orm-pack``.
+Symfony Flex will download recipes and install all necessary configuration and entity class for notification messages.
```
Comment (greg0ire) [inline docs/reference/installation.rst]: Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Comment (greg0ire) [inline docs/reference/installation.rst]: entity classes? entities?

Comment (covex-nn) [reply docs/reference/installation.rst]: there is only one entity class `App\Entity\SonataNotificationMessage`

Comment (greg0ire) [reply docs/reference/installation.rst]: :thinking: … maybe this then: "Symfony Flew will download recipes and install all necessary configuration files, and an entity class for notification messages" ?

Comment (covex-nn) [reply docs/reference/installation.rst]: Done. Thank you for fixing me )

**Comment by covex-nn** [comment]
@kunicmarko20 i added a note to `Extending the Bundle` section

**Comment by greg0ire** [comment]
Please kindly squash your commits together. If you don't, we'll try to remember to do it for you but it's best if you save us this trouble.

How to do that?

1. `git rebase -i origin/3.x`, assuming `origin` is a git remote that points to this repository, and not your fork. If you're not sure what your remotes are, run `git remote -vvv`, there should be your fork and the holy/reference/base/origin/whatever-you-call-it repository.
2. A window will show up with many lines, replace `pick` with `fixup` on every line but the first one
3. Close your editor, git should do its magic, and you should end up with one commit
4. Use `git push --force` to overwrite what you already pushed. Don't forget the `--force` option otherwise git will try to merge both things together.

**Comment by greg0ire** [comment]
Thanks @covex-nn !

**Pull request merged**